src/sentry/api/endpoints/project_details.py

```diff
@@ -270,7 +264,6 @@ def validate_slug(self, slug: str) -> str:
             raise serializers.ValidationError(
                 "Another project (%s) is already using that slug" % other.name
             )
-        slug = super().validate_slug(slug)
```

Comment:
Is this not needed anymore because you're validating with the SentrySlugField?

Reply:
yup we're validating with the regex itself instead of `validate_slug` that was on the mixin